diffblue-base.yml

```diff
@@ -10,3 +10,4 @@
 # be overriden to keep consistency with other benchmarks.
 buildCmd: cd src/supernode; mvn compile
 testCmd: cd src/supernode; mvn test
+classpathCmd: cd src/supernode; mvn dependency:build-classpath -Dmdep.includeScope=runtime -Dmdep.outputFile=/dev/fd/4 4>&1 >/dev/null | sed "s/:/\\n/g"
```

Comment:
No idea what there are:
`-Dmdep.includeScope=runtime` `-Dmdep.outputFile=/dev/fd/4`

What's the default classpath command for other projects?

Reply:
`mvn dependency:build-classpath -Dmdep.includeScope=runtime -Dmdep.outputFile=/dev/fd/4 4>&1 >/dev/null | sed "s/:/\\n/g"` is the new default classpath command.
We changed it because the previous classpath command depended on grepping the output of the `buildCmd` and it didn't work if `maven-compiler-plugin` wasn't in the `pom.xml` for the project.

`-Dmdep.includeScope=runtime` tells maven that we're only interested in the runtime dependencies and not compile ones.
`-Dmdep.outputFile=/dev/fd/4` is part of a bit of redirection magic so that we only get the classpath and not the other build status messages that maven likes to print to stdout.

Reply:
@allredj See https://github.com/diffblue/platform/pull/2045

Reply:
If what you're adding is the default classpath used for all projects, can you please add a comment into the file about that? It's useful information in case something goes wrong or the default changes.